### DISCUSSION TITLE
GIF drawing on your profile readme

### DISCUSSION THREAD

**Discussion by devtahmid** [mrhrifat/mrhrifat#10] [Q&A] [answered]
Assalamua a'laikum @mrhrifat !
Is there a tutorial to make an illustration like you have on your profile readme(the one with a coder in front of a computer)? If it was a commission, is it alright if I ask who made it? It looks really good

**Comment by mrhrifat** [comment, marked as answer]
Wa'alaikumus Salam... I don't know the creator or the owner of this GIF illustration. You can search on many website who provide illustration like, `freepik, freevector, vectorstock`. I hope you'll find it there with the owner or the creator name or profile.
